lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -141,6 +145,19 @@
   }
 }
 
+.nx-dropdown-link .nx-icon {
+  &:hover {
+    color: var(--nx-color-link-hover);
+    text-decoration: underline;
```

Comment:
In the dropdown mocks, the external link icons are not underlined.

Reply:
> In the dropdown mocks, the external link icons are not underlined.

Removed: 602eb123d4617a696be4b955b4cdb781fd26ed1b